ITR/interfaces.py

```diff
@@ -488,7 +488,7 @@ class ICompanyData(BaseModel):
 
     sector: str  # TODO: make SortableEnums
     region: str  # TODO: make SortableEnums
-    target_probability: float = 0.5
```

Comment:
Here is where we should initialize the probability to non-existent.  If we use the datatype float, then that means np.nan.  If we want to use pd.NA, we can use pd.Float64Dtype as the datatype.  But if we initialize to a non-empty value, it would not be overridden in the calculation of the temperature score (given the above).

Reply:
In ICompany the probability is now initialized to non-existent and later missing values are replaced by the default value.